src/main/java/com/flansmod/common/FlansMod.java

```diff
@@ -668,7 +669,9 @@ public static void syncConfig() {
         TeamsManager.bulletSnapshotDivisor = configFile.getInt("BltSS_Divisor", "Teams/advanced settings", 50, 0, 1000, "Divisor");
 
         //Server/Gameplay Settings (Server-client synced)
-        enableKillMessages = configFile.getBoolean("enableKillMessages", "Gameplay Settings (synced)", enableKillMessages,"Enable killMessage display");
+        globalBulletSpeedMultiplier = configFile.getFloat("globalBulletSpeedMultiplier", "Gameplay Settings (synced)", 1.0F, 0F, 100F, "Set the global bullet speed multiplier");
```

Comment:
All three of these also need to be added to PacketModConfig for syncing - including the kill message one